doc/Data Plane Code Generation/Subclients.md

```diff
@@ -0,0 +1,233 @@
+# Customizing the generated code
```

Comment:
It may be worth pulling in information from this document as well: https://github.com/Azure/autorest.csharp/blob/feature/v3/docs/features/subclients.md

We may need to bring it up to date, but it provides a nice description.